### PR TITLE
chore(): update comment

### PR DESCRIPTION
**Pull request by cgundy**
# Motivation

This updates the comment with the currently correct information.

# Changes

No code changes, just updating a comment.

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ jobs:
...
     runs-on: ubuntu-20.04
 
-    # In order to trigger the CLA after committing formatting changes, we need
-    # to use the GIX_CREATE_PR_PAT token. This token is not available for all
-    # users. So on PRs where the token is not available, we don't commit
-    # changes and instead just fail if the formatting is incorrect.
+    # In order to trigger other workflows after committing docs changes, we need
+    # to use the PR Automation App. This token is not available for external
+    # contributors. So on forked PRs, we don't commit changes and instead just fail 
+    # if the docs changes are needed.
```
Comment (dskloetd) [inline .github/workflows/checks.yml:16]: again "docs" instead of "formatting"?

File: .github/workflows/checks.yml
Comment (dskloetd) [inline .github/workflows/checks.yml:1]: GitHub won't let me comment on line 73 so I'll make a file comment instead.

On line 73, it says "personal access token". Is that still correct?

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -10,10 +10,10 @@ jobs:
   formatting:
     runs-on: ubuntu-20.04
 
-    # In order to trigger the CLA after committing formatting changes, we need
-    # to use the GIX_CREATE_PR_PAT token. This token is not available for all
-    # users. So on PRs where the token is not available, we don't commit
-    # changes and instead just fail if the formatting is incorrect.
+    # In order to trigger other workflows after committing docs changes, we need
+    # to use the PR Automation App. This token is not available for external
```
[the remark this answers is not in the document]
Comment (cgundy) [reply .github/workflows/checks.yml:14]: Good point, I'll change it to "this secret".

File: .github/workflows/checks.yml [hunk 3]
```diff
@@ -10,10 +10,10 @@ jobs:
...
+    # contributors. So on forked PRs, we don't commit changes and instead just fail
```
[the remark this answers is not in the document]
Comment (cgundy) [reply .github/workflows/checks.yml:15]: Ah wow, good memory! Should i just be very explicit then and say "forked PR or from dependabot"?

[the remark this answers is not in the document]
Comment (dskloetd) [reply .github/workflows/checks.yml]: Is dependabot really the only other possibility?
Isn't there another way to do what dependabot does?
Why not just stick with "PRs where the app secret is not available"?

Even though I guessed correctly what was meant by "forked PR", I do find that phrase a bit confusing.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**